doc/Tutorials/Introduction.ipynb

```diff
@@ -550,7 +550,7 @@
    "source": [
     "print(rgb_parrot)\n",
     "print(rgb_parrot[0,0])\n",
-    "print(rgb_parrot[0,0][0])"
+    "print(rgb_parrot[0,0].iloc[0, 0])"
```

Comment:
This was left over from when we returned tuples when indexing RGBs, so I ended up updating it, suppose ``rgb_parrot[0, 0, 'R']`` would have been clearer but we're probably throwing this notebook out right?